torchdata/datapipes/iter/util/converter.py

```diff
@@ -26,6 +26,8 @@ class IterToMapConverterMapDataPipe(MapDataPipe):
     with exactly two objects. The first object of each item becomes a key in
     the new dictionary, and the second object the corresponding value.
 
+    For the opposite converter, use ``MapToIterConverter``.
```

Comment:
IIRC, we can do something like `:class:torchdata.datapipes.map.MapToIterConverter`